apply.js

```diff
@@ -67,6 +67,13 @@ const patches = [
       css: ["hideHelpOnDashboard.css"],
     },
   },
+  {
+    name: "Custom font",
+    description: 'Applies default OS font',
```

Comment:
It's not even a sentence :sob:, pls change name and description to ones actually explaining what does it do